### PR TITLE
aruha-998: release version

### PR DESCRIPTION
**Pull request by adyach**
aruha-998

[ARUHA-XXX: Name of ticket](link to ticket)

## Description
A few sentences describing the overall goals of the pull request's
commits.

## Review
- [ ] Tests
- [ ] Documentation
- [ ] CHANGELOG

## Deployment Notes
These should highlight any db migrations, feature toggles, etc.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.2.2] - 2017-09-28
```
Comment (v-stepanov) [inline CHANGELOG.md]: To say the truth for me it feels like release of a new feature, so it could probably be 2.3.0
Or is it already published as 2.2.2 ?

Comment (adyach) [reply CHANGELOG.md]: I can rename it in release notes and in change log, but the tag name still will be 2.2.2, okay?

Comment (adyach) [reply CHANGELOG.md]: I can even create a new tag )

Comment (v-stepanov) [reply CHANGELOG.md]: No, I'm totally fine with 2.2.2
I think there is now need to redo this.

**Comment by v-stepanov** [comment]
👍

**Comment by adyach** [comment]
👍

**Pull request merged**